src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckPolicyMessageContentCollector.java

```diff
@@ -62,6 +70,63 @@ public BlackDuckPolicyMessageContentCollector(final JsonExtractor jsonExtractor,
 
     @Override
     protected void addCategoryItems(final List<CategoryItem> categoryItems, final JsonFieldAccessor jsonFieldAccessor, final List<JsonField<?>> notificationFields, final NotificationContent notificationContent) {
+        if (NotificationType.POLICY_OVERRIDE.name().equals(notificationContent.getNotificationType())) {
```

Comment:
Should we split this into two Collectors? PolicyOverride and PolicyRule

Reply:
We should.  Given that we are trying to release the next version I would want to limit the changes in this version and enhance master with changes to create a separate collector for policy overrides.  Limit the number of changes in this branch.